src/include/catalog/pg_proc.dat

```diff
@@ -8547,6 +8547,9 @@
 { oid => '2730', descr => 'trigger description with pretty-print option',
   proname => 'pg_get_triggerdef', provolatile => 's', prorettype => 'text',
   proargtypes => 'oid bool', prosrc => 'pg_get_triggerdef_ext' },
+{ oid => '9569', descr => 'get CREATE statement for a trigger',
+  proname => 'pg_get_trigger_ddl', proisstrict => 't', prorettype => 'text',
```

Comment:
@alvherre for this one, I put ``proisstrict => 't'`. I noticed that for other functions like `pg_get_triggerdef` it does not use this. Because it doesn't set `proisstrict`, it has to handle NULL using:
```
if (res == NULL)
		PG_RETURN_NULL();
```
Question: Could we create another patch for `pg_get_triggerdef` and `pg_get_triggerdef_ext` to use `proisstrict => 't'`?